docs/source/commandline/commandline-all.rst

```diff
@@ -928,95 +1148,176 @@ Core Video Options
+      |
+      |	Using ``none`` displays no windows and does no drawing.  This is primarily present for doing CPU benchmarks without the overhead of the video system.
+      |
+
+    On Windows:
+
+      |	Using ``gdi`` tells MAME to render video using older standard Windows graphics drawing calls.  This is the slowest but most compatible option on older versions of Windows.
+      |
+      |	Using ``d3d`` tells MAME to use Direct3D for rendering.  This produces the better quality output than ``gdi`` and enables additional rendering options.  It is recommended if you have a semi-recent (2002+) video card or onboard Intel video of the HD3000 line or better.
+      |
+
+    On other platforms (including SDL on Windows):
+
+      |	Using ``accel`` tells MAME to render video using SDL's 2D acceleration if possible.
+      |
+      |	Using ``soft`` uses software rendering for video output.  This isn't as fast or as nice as OpenGL but will work on any platform.
+      |
+
+    Defaults:
+
+      |	The default on Windows is ``d3d``.
+      |
+      |	The default for Mac OS X is ``opengl`` because OS X is guaranteed to have a compliant OpenGL stack.
+      |
+      |	The default on all other systems is ``soft``.
+      |
 
 .. _mame-commandline-numscreens:
 
 **-numscreens** *<count>*
 
-	Tells MAME how many output windows to create. For most systems, a single output window is all you need, but some systems originally used multiple screens (*e.g. Darius and PlayChoice-10 arcade machines*). Each screen (up to 4) has its own independent settings for physical monitor, aspect ratio, resolution, and view, which can be set using the options below. The default is ``1``.
+    Tells MAME how many output windows to create.  For most systems, a single
+    output window is all you need, but some systems originally used multipl
+    screens (*e.g. Darius and PlayChoice-10 arcade machines*).  Each screen
+    (up to 4) has its own independent settings for physical monitor, aspect
+    ratio, resolution, and view, which can be set using the options below.
+
+    The default is ``1``.
 
 .. _mame-commandline-window:
 
 **-[no]window** / **-[no]w**
 
-	Run MAME in either a window or full screen. The default is OFF (**-nowindow**).
+    Run MAME in either a window or full screen.
+
+    The default is OFF (**-nowindow**).
 
 .. _mame-commandline-maximize:
 
 **-[no]maximize** / **-[no]max**
 
-	Controls initial window size in windowed mode. If it is set on, the window will initially be set to the maximum supported size when you start MAME. If it is turned off, the window will start out at the smallest supported size. This option only has an effect when the -window option is used. The default is ON (**-maximize**).
+    Controls initial window size in windowed mode.  If it is set on, the window
+    will initially be set to the maximum supported size when you start MAME.  If
+    it is turned off, the window will start out at the smallest supported size.
```

Comment:
`-nomax` is supposed to cause the window to be sized so that emulated screens are not being shrunk in any dimension, i.e. if there's a single screen with square pixels it will be 1:1 emulated to host pixels, and with non-square pixels it'll be enlarged on one axis only.

Reply:
Think I have this worked out.